packages/near-fast-auth-signer/src/components/Sign/Sign.styles.ts

```diff
@@ -7,127 +7,143 @@ interface ModalSignWrapperProps {
 
 export const ModalSignWrapper = styled.div<ModalSignWrapperProps>`
   ${(props) => props.hide && 'opacity: 0;'}
+  max-height: 660px;
   width: 550px;
   margin: 0 auto;
   border-radius: 8px;
   padding: 24px;
   display: flex;
   flex-direction: column;
   position: relative;
-  row-gap: 8px;
+  overflow: hidden;
+
+  .modal-body {
+    height: 100%;
+    width: 100%;
+    overflow: scroll;
```

Comment:
replace this to `auto` so that when it is collapsed, UI looks better as scroll is not needed. 
If you change this to auto:
<img width="467" alt="image" src="https://github.com/near/fast-auth-signer/assets/6027014/c394ddd0-1bf4-4780-93f4-127500b9a97a">

Original code:
<img width="521" alt="image" src="https://github.com/near/fast-auth-signer/assets/6027014/f96e773f-aed7-46d4-9c4a-f21aa3a2f65e">

Reply:
Good point, I will replace it :)